notebooks/tutorials/coarse_grained_polymer.ipynb

```diff
@@ -0,0 +1,487 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining the force field\n",
+    "We will cover two methods for defining the forcefield:\n",
+    "\n",
+    "1. Using the Python API.\n",
+    "2. Creating a forcefield xml file.\n",
+    "\n",
+    "Both methods achieve the same result of defining the forcefield. There are pros and cons of both ways. The key points are that using the Python API is more flexible, while creating a forcefield xml file makes it easier to share your forcefield and helps reproducibility. This is demonstrated in the other OpenMM tutorials --- the standard forcefields such as Amber are distributed with OpenMM as xml files, while the custom forces used to do free energy calculations are defined using the Python API."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining a force-field using the Python API\n",
+    "\n",
+    "We can create instances of the OpenMM [Force classes](http://docs.openmm.org/latest/api-python/library.html#forces), assign parameters, and add them to the system.\n",
+    "First we must create a [System](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.System.html#openmm.openmm.System).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create the system and add the particles to it\n",
+    "system = mm.System()\n",
+    "system.setDefaultPeriodicBoxVectors(*topology.getPeriodicBoxVectors())\n",
+    "for atom in topology.atoms():\n",
+    "    system.addParticle(atom.element.mass)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will then create a [HarmonicBondForce](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.HarmonicBondForce.html#openmm.openmm.HarmonicBondForce) and a [CustomNonBondedForce](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.CustomNonbondedForce.html#openmm.openmm.CustomNonbondedForce). \n",
+    "\n",
+    "For each bond that is defined in the topology we need to add it to the `HarmonicBondForce`. We set the equilibrium bond lengths and spring constants to be 0.38 nm and 1000 kJ/mol/nm respectively which are consistent with parameters used in amino-acid level CG models [1]. \n",
+    "\n",
+    "We define the `CustomNonBondedForce` to be a Lennard-Jones interaction. We set the value of sigma to be 0.5nm to approximate the size of an amino acid and the value of epsilon to 1 kJ/mol to create a potential that is attractive at 300K. Please do not use these parameters for real simulations. You should consult the literature and chose an appropriate CG force-field or systematically create your own parameter set! Finally, we add exclusions between atoms that are directly bonded and set a cutoff of 1.5nm."
```

Comment:
Why use a CustomNonbondedForce rather than NonbondedForce?  There's nothing wrong with using the custom force.  I'm just curious, since you create a custom force with exactly the same functional form as the corresponding standard force.

Reply:
I will add a note that the standard force could be used but the tutorial is demonstrating the flexibility of customNonBonded. People typically use different potentials such as screened coulomb terms and want full control of the combination rules.